app/PostStatus.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class PostStatus extends Model
+{
+    protected $table = 'post_statuses';
+    public $timestamps = false;
+    protected $guarded = [];
```

Comment:
I guess it's default value of Eloquent model. Do not include default values.

Reply:
Please remove it everywhere

Reply:
Which exactly?
As I read guarded is needed to allow mass assignment and timestamps, cause we do not have these fields for that table.

Reply:
It's better to use politic of what "all denied except allowed". Use `$fillable` property instead of `$guarded`. Otherwise there is a risk to miss some field after adding it in future from "guardening".